docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,529 @@
+        namespace: backend-ns
+      default:
+        connectionTimeout: 5s
+```
+
+Important to note, even though policy in `backend-ns` changes proxy configurations in `frontend-ns` 
+the top-level targetRef still servers the purpose of showing what proxy configurations are going to change.
+Policy's namespace **does not** implicitly turn the top-level targetRef to `MeshSubset{tags:{"kuma.io/namespace":"backend-ns"}}`.
+
+### Namespace-scoped policies and multizone
+
+Namespace-scoped policies can be applied to a custom namespace only on zones as there are simply no DPPs and custom
+namespaces on Global.
+
+After a user applied a policy on zone's custom namespace, the webhook automatically adds a `k8s.kuma.io/namespace` label to the policy.
+The label is part of the hash suffix when policy is synced to global.
+
+As mentioned previously, a policy applied in producer namespace can affect envoy configs of consumers.
+Consumers can reside not only in different namespaces, but also in different zones.
+That's why, the policy applied in `backend-ns` in `zone-1` has to be synced to global and then to other zones.
+With the increased blast radius of the policies comes great responsibility, 
+we don't want small mistake in the policy to break traffic on the other side of the world.
+A few things could be done to protect users from impactful mistakes:
+* Enhanced validation. If policy is created in `backend-ns` of the `zone-1` and the top-level targetRef is `Mesh`, 
+then `to[].targetRef` should have both `k8s.kuma.io/namespace: backend-ns` and `kuma.io/zone: zone-1`. 
+* Ability to opt-in or opt-out from syncing the policy to other zones.
+
+#### Enhanced validation
+
+Since applying a policy to the custom namespace of a zone has a potential to change envoy configs of all pods in the mesh including pods in other zones,
+we need to restrict "to" section, so it affect only the requests going to the custom namespace.
+For example, the following policy when applied to `backend-ns` is too intrusive toward the backend consumers:
+
+```yaml
+kind: MeshTimeout
+metadata:
+  namespace: backend-ns
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        labels:
+          kuma.io/display-name: redis
+          k8s.kuma.io/namespace: redis-ns
+      default:
+        connectionTimeout: 5s # why does the Backend Service Owner try to change the way clients use the 'redis' service?
+```
+
+Such policy should not be allowed, if Backend Service Owner needs to change the way clients use `redis`
+they have to put the policy into `kuma-system` namespace by checking with Mesh Operator first.
+
+The validation rules for namespace-scoped policy are the following:
+* to-policy when placed into the custom namespace must have the namespace and zone either in the top-level targetRef or in each to-item
+* from-policy when placed into the custom namespace must always have the namespace and zone in the top-level targetRef
```

Comment:
That seems unnecessarily annoying. Again UX is important here!

Reply:
Isn't `from` policy on custom namespace having an "implicit MeshSubset" a better behaviour?  Especially if we go for https://github.com/kumahq/kuma/issues/10247 top level targetRef slowly disappears.

Reply:
Also with the current zone originated policies is similar to having an implicit `MeshSubset` no?